### DISCUSSION TITLE
Система кассет

### DISCUSSION THREAD

**Discussion by SnakeLuck** [octothorp-team/public-dobrograd#720] [Предложения]
Катаясь на машине друга я обнаружил что радио транслирует радио станции (что логично), но тогда почему нет возможности включить что-то новенькое, необычное ну и просто свое а не слушать радиостанцию в надежде на то что ваш любимый трек появится в эфире. Ну и это нелогично что нельзя запустить кассету с твоей любимой песней в машине, а бред по типу отыгрыша того что ты вставил кассету а потом включил музыку в соунд паде и потом всю поездку звучит абсолютно тупо, так и по моему нонрпешно. и я предлагаю добавить систему кассет для машины. Как я это вижу? в С меню в разделе авто будет еще один раздел под "Радио" и название ему "Кассеты" и при нажатии туда у вас всплывает окно в которое нужно вставить ссылку на ютуб страницу на которой находится нужная для проигрывания музыка. Для полицейских машин эту функцию стоит не добавлять по ясным причинам. И того у нас появится удобная для любителей своей музыки фича. Спасибо за внимание!

**Comment by Dawaren** [comment]
Это хорошая идея, не спорю. Но явно будут люди, которые будут включить полную парашу на всю громкость, увы :/

**Reply by SnakeLuck** [reply]
Для этого существуют жалобы и регулятор громкости, или же можно поставить блокировку на громкость особям которые будут устраивать всякую дичь. Так что не очень критично, и плюс я видел не очень много полных кретинов которые катаются на машине, не спорю они есть, но их мало.

**Comment by Leygun** [comment]
Привет! Я, как радио модератор, хочу тебе  сказать, что такого точно не будет, каждый трек должен проходить проверку. У нас есть свои определенные правила, которые не позволяют ставить любую музыку, которую ты захочешь.

**Reply by SnakeLuck** [reply]
эххх. Неприятно :(
